_ml-commons-plugin/cluster-settings.md

```diff
@@ -12,32 +12,30 @@ This page provides an overview of `opensearch.yml` settings that can be configur
 
 ## Run tasks and models on ML nodes only
 
+If `true`, ML Commons tasks and models run machine learning (ML) tasks on ML nodes only. If `false`, tasks and models run on ML nodes first. If no ML nodes exist, tasks and models run on data nodes. Don't set as "false" on production cluster. 
+
 ### Setting
 
 ```
 plugins.ml_commons.only_run_on_ml_node: true
 ```
 
-### Description
-
-If `true`, ML Commons tasks and models run machine learning (ML) tasks on ML nodes only. If `false`, tasks and models run on ML nodes first. If no ML nodes exist, tasks and models run on data nodes. Don't set as "false" on production cluster. 
-
 ### Values
 
 - Default value: `false`
 - Value range: `true` or `false`
 
 ## Dispatch tasks to ML node 
 
+`round_robin` dispatches ML tasks to ML nodes using round robin routing. `least_load` gathers all an ML nodes' runtime information, like JVM heap memory usage and running tasks, then dispatches tasks to the ML node with the least load.
```

Comment:
all an ML --> all ML
```suggestion
`round_robin` dispatches ML tasks to ML nodes using round robin routing. `least_load` gathers runtime information from all ML nodes, like JVM heap memory usage and running tasks, then dispatches the tasks to the ML node with the least load.
```

Reply:
Should the last two words be "lowest load"? low - lower - lowest; little - less/lesser - least